xml/Microsoft.Extensions.Hosting/HostingAbstractionsHostExtensions.xml

```diff
@@ -50,7 +50,7 @@
       </Parameters>
       <Docs>
         <param name="host">The <see cref="T:Microsoft.Extensions.Hosting.IHost" /> to run.</param>
-        <summary>Runs an application and blocks the calling thread until host shutdown.</summary>
+        <summary>Runs an application and blocks the calling thread until host shutdown is triggered and all <see cref="T:Microsoft.Extensions.Hosting.IHostedService" /> instances are stopped.</summary>
```

Comment:
The issue you linked is for `RunAsync` not `Run`. Let's only update the doc over `RunAsync`.

Do you have a code repro showing the same holds true for `Run`?

cc @eerhardt

Reply:
Run calls RunAsync

https://github.com/dotnet/runtime/blob/2be87c9c976c0cd21b66ea12ae406a559aacaac0/src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/HostingAbstractionsHostExtensions.cs#L44-L51